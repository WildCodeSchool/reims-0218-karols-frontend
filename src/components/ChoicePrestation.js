import React from "react"
import { Card, CardTitle, CardText, CardImgOverlay } from "reactstrap"

const ChoicePrestation = () => {
  return (
    <div className="mb-4">
      <Card inverse>
        <img
          className="img-fluid"
          src="https://images.unsplash.com/photo-1520173043194-dc6b2a237fee?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=9da78404d02fafbd40afdf2b59dc2d9d&auto=format&fit=crop&w=800&q=60"
          alt="Prestations"
        />
        <CardImgOverlay>
          <CardTitle>Card Title</CardTitle>
          <CardText>Choisissez votre prestation</CardText>
        </CardImgOverlay>
      </Card>
    </div>
  )
}

export default ChoicePrestation
