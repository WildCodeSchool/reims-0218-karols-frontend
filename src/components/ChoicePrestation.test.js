import React from "react"
import ReactDOM from "react-dom"
import ChoicePrestation from "./ChoicePrestation"
import renderer from "react-test-renderer"

it("renders without crashing", () => {
  const div = document.createElement("div")
  ReactDOM.render(<ChoicePrestation preparations={[]}/>, div)
  ReactDOM.unmountComponentAtNode(div)
})

it("renders correctly", () => {
  const tree = renderer.create(<ChoicePrestation preparations={[]}/>).toJSON()
  expect(tree).toMatchSnapshot()
})
