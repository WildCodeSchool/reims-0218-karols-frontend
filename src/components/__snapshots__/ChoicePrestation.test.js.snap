// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    className="card text-white"
  >
    <img
      alt="Prestations"
      className="img-fluid"
      src="http://via.placeholder.com/250x250"
      width="250"
    />
    <div
      className="card-img-overlay"
    >
      <h5
        className="card-title"
      >
        Card Title
      </h5>
      <p
        className="card-text"
      >
        Choisissez votre prestation
      </p>
    </div>
  </div>
</div>
`;
